Adafruit_GFX.h

```diff
@@ -286,7 +287,7 @@ class Adafruit_GFX_Button {
   uint8_t       _textsize_x;
   uint8_t       _textsize_y;
   uint16_t      _outlinecolor, _fillcolor, _textcolor;
-  char          _label[10];
+  char          *_label;
```

Comment:
used to be 10 bytes. Now it's 4 bytes for the pointer, plus 4 bytes for the internal allocated array size, plus 4 bytes allocator node overhead plus alignment overhead. So 12 of overhead PLUS the length of the label itself, so a 9-char button goes up from 10 to 28 bytes. Maybe just change the 10 to something bigger if you need something bigger. Maybe make it a static constant to be tweaked. Button labels aren't going to get that big.